packages/react/fast-components-react-base/package.json

```diff
@@ -112,7 +112,6 @@
     "ts-jest": "^25.4.0",
     "ts-loader": "^4.0.1",
     "typescript": "3.3.3",
-    "utility-types": "^2.1.0",
```

Comment:
I don't think utility-types is needed now that we've updated TypeScript. I think we can simply remove it.